package-res/resources/web/pentaho/visual/ccc/abstract/model.js

```diff
@@ -196,14 +194,63 @@ define([
             name: "emptyCellMode",
             type: emptyCellModeFactory,
             required: true,
-            value: "GAP"
+            value: "gap"
           },
 
           { // HG, Scatter
             name: "sizeByNegativesMode",
             type: sizeByNegativesModeFactory,
             required: true,
-            value: "NEG_LOWEST"
+            value: "negLowest"
+          },
+
+          //region Pattern, Color Set and Reverse Colors
+
+          //Used by Scatter and HeatGrid
+          {
+            id: "pattern",
```

Comment:
Do not declare `required` without a default value! The same goes to the following.